src/components/Hero.tsx

```diff
@@ -37,14 +37,14 @@ const Hero = () => {
   
 
   return (
-    <div className='flex flex-col my-[95px] items-center'>
+    <div className='flex flex-col py-[95px] items-center mx-[10px] '>
         <div>
-          <h1 className='font-Lexend text-[48px] text-center text-lightSlate leading-[52.8px] tracking-[-6%] '>
+          <h1 className='font-Lexend text-[40px] md:text-[48px] text-center text-lightSlate leading-[52.8px] tracking-[-6%] '>
```

Comment:
Updating the text size, leading, and tracking.

Here's a suggestion:

```suggestion
          <h1 className='font-Lexend text-4xl md:text-5xl text-center text-lightSlate leading-tight tracking-tight '>
```

Although Figma uses percentages for `letter-spacing` (`tracking`), it doesn't work on the browser... so we have to use `rem` units. `tracking-tight` has a value of `-0.25rem` 👌